samba/CHANGELOG.md

```diff
@@ -1,4 +1,9 @@
 # Changelog
+## 12.4.1
+
+- Add the option to enable SAMBA to become local browsermaster
+- Add the option to enable SAMBA WINS Server
```

Comment:
As discussed in the other review, since WINS is really old and deprecated, let's remove the WINS option from this PR.